# branch
111111111111111111111111
master
commit


## test

```
	Cause: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: 
		The last packet successfully received from the server was 53,874,730 milliseconds ago.  
		The last packet sent successfully to the server was 53,874,730 milliseconds ago. 
		is longer than the server configured value of 'wait_timeout'. 
		You should consider either expiring and/or testing connection validity before use in your application, 
		increasing the server configured values for client timeouts, 
		or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.
```